src/apprt/gtk/Surface.zig

```diff
@@ -1264,6 +1265,90 @@ pub fn showDesktopNotification(
     c.g_application_send_notification(g_app, body.ptr, notification);
 }
 
+/// Handle bell features.
+pub fn bell(self: *Surface) void {
+    inline for (std.meta.fields(configpkg.Config.BellFeatures.Features)) |field| {
+        const feature = std.meta.stringToEnum(configpkg.Config.BellFeatures.Features, field.name) orelse unreachable;
+        const enabled = @field(self.app.config.@"bell-features", field.name);
+        if (enabled) {
+            switch (feature) {
```

Comment:
Honestly I think we should just do it the "dumb" way and use individual if clauses... This is a bit too hard to read even for me

Reply:
Fixed.